docs/style.css

```
@@ -9,36 +9,66 @@
  * No glassy backdrops, no floaty shadows, sharp warm borders, generous space.
  * ─────────────────────────────────────────────────────────────────────────── */
 
+/* Geist Mono — docs.json fonts only accepts heading+body, so the code face
+   is loaded here. */
+@import url("https://fonts.googleapis.com/css2?family=Geist+Mono:wght@400;500&display=swap");
```

Stylelint (17.12.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Geist+Mono:wght@400;500&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Geist+Mono:wght@400;500&display=swap"" (import-notation)

(import-notation)